code/run_capsule.py

```diff
@@ -49,7 +50,7 @@ def get_input_parser() -> argparse.ArgumentParser:
     parser.add_argument("--use_data_asset_csv", type=int, default=0)
     parser.add_argument("--file_extension", type=str, default="")
     parser.add_argument("--split_files", type=int, default=1)
-    parser.add_argument("--max_jobs", type=int, default=50)
+    parser.add_argument("--tasks_per_job", type=int, default=1)
     parser.add_argument("--group_by", type=str, default="",)
 
```

Comment:
we should really switch to using pydantic settings to parse here at some point, either in this PR or later